CLI/astutil.go

```diff
@@ -65,6 +97,20 @@ func nodeToVec(n node, size int, name string) ([]float64, error) {
 	return utils.ValToVec(val, size, name)
 }
 
+func nodeToColorString(colorNode node) (string, error) {
+	colorInf, err := colorNode.execute()
+	if err != nil {
+		return "", err
+	}
+
+	color, ok := utils.ValToColor(colorInf)
+	if !ok {
+		return "", fmt.Errorf("Please provide a valid 6 digit Hex value for the color")
+	}
+
+	return color, nil
+}
+
```

Comment:
I had implemented similar checks before but we decided with helder that it was better to remove them because it is supposed to be implemented on the API side already, so it would make it harder to maintain because you have to keep both sides in sync.

Reply:
I don't quite understand this comment because it is in the line that says "return color, nil" and I suspect you were referring to another part of the code.

Reply:
Sorry for the misselection, I was talking about the regex check right above for the slug. I thought you also added one for the color, but apparently there was already a function checking that the color is valid, but I think it should be removed because it is also checked on the API side.

Reply:
Ok now i see. Fixed in caf10df1. Only now instead of having a clear error the error is:
Error: [Response From API] JSON body doesn't validate with the expected JSON schema
    /slug does not match pattern '^[a-z0-9-_]+$'

But I think it is a more general problem than with slugs, I will create an issue to improve the error message in these cases.